README.md

```diff
@@ -121,11 +121,11 @@ Finally you need to install the Integration into the repo. Do this by going to y
 
 ### Tailoring ProcBots for a Repo via Configuration File
 
-ProcBots now also respond to a configuration file. This is a file with the name `.procbots.yml` in a relative location for the ProcBot running. In the case of the VersionBot, this is in the root of the repository that it is working on.
+ProcBots now also respond to a configuration file. This is a file with the name `.procbots.yml` in a relative location for the ProcBot running. In the case of the GithubBot (and any other child bot derived from it, such as VersionBot), this is in the root of the repository that it is working on, on the `master` branch.
 
-The configuration file uses a set of nested properties based on the class hierarch of the ProcBots, with each class able to modify variables at run time from the configuration file, eg:
+The configuration file uses a set of nested properties based on the class hierarchy of the ProcBots, with each class able to modify variables at run time from the configuration file.
 
-ProcBots itself has a single property, `minimum_version`, which is checked to ensure that operations only get carried out should that version be satisfied.
+ProcBots itself has a single property, `minimum_version`, which is checked to ensure that operations only get carried out should that version be satisfied. Should it not find itself to be of at least the minimum version, and error is thrown.
```

Comment:
Hmm, and I wonder if it'd be worth extending this to the "child classes" too, so that GithubBot and VersionBot in `.procbots.yml` could each specify their `minimum_version`? (although obviously at this early stage it's maybe a bit overkill)